src/platform/unix.rs

```diff
@@ -9,13 +9,20 @@
   target_os = "openbsd"
 ))]
 
+use std::{os::raw::c_int, sync::Arc};
+
+// XConnection utilities
+pub use crate::platform_impl::x11;
```

Comment:
Should be hidden from docs.

```suggestion
#[doc(hidden)]
pub use crate::platform_impl::x11;
```